### ISSUE TITLE
Document communication protocol

### ISSUE THREAD

**Issue by SamMousa** [Roeshambo/MonolithDKP#300]
It would be nice if communication between agents using the addon is documented somewhere.

This could open the door for more contributors as well as allowing other addons to interact with Monolith agents.

**Comment by Roeshambo** [comment]
Are you referring simply to a write up identifying which prefixes are sending what data as well as what parameters are required?

**Comment by SamMousa** [comment]
Yes, ideally as a [Message Sequence Chart](https://en.wikipedia.org/wiki/Message_sequence_chart)

That way it is a lot easier to understand and we could start to modularize the code as well.
Thinks like a bidding window should be a totally separate module (possibly even a separate addon).

When this is documented it will make the code easier to understand for potential contributors!

You could use something like this: https://github.com/mermaid-js/mermaid, if you prefer to describe in words I can make the charts / docs out of them!
